src/components/Elevation/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+import useStyles from './styles';
+
+export interface IElevationProps {
+  variant?: 'extraLight' | 'light' | 'medium' | 'heavy' | 'extraHeavy';
+  margin?: TextStyle['margin'];
+  width?: TextStyle['width'];
+  height?: TextStyle['height'];
+  justifyContent?: TextStyle['justifyContent'];
+  alignItems?: TextStyle['alignItems'];
+  borderRadius?: TextStyle['borderRadius'];
```

Comment:
might be just "justify, align and radius". I have seen such usage. Not a big case though :)